pkg/agent/openflow/client.go

```diff
@@ -342,9 +342,17 @@ type Client interface {
 	// InstallMulticlusterClassifierFlows installs flows to classify cross-cluster packets.
 	InstallMulticlusterClassifierFlows(tunnelOFPort uint32, isGateway bool) error
 
-	// UninstallMulticlusterFlows removes cross-cluster flows matching the given clusterID on
+	// InstallMulticlusterPodFlows installs flows to handle cross-cluster packets from Multi-cluster Gateway to
+	// regular Nodes.
+	InstallMulticlusterPodFlows(podIP net.IP, tunnelPeerIP net.IP) error
+
+	// UninstallMulticlusterFlows removes cross-cluster flows matching the given cache key on
 	// a regular Node or a Gateway.
-	UninstallMulticlusterFlows(clusterID string) error
+	UninstallMulticlusterFlows(key string) error
```

Comment:
Looked at other Uninstall funcs. They do not use "key string" as the argument, but a meaningful name to indicate what is the key (e.g. hostInterfaceName, ofPort, svcIP, etc.). Could we follow the same way?

Reply:
Done